src/tools/simple-kernel-timer/kp_kernel_info.h

```diff
@@ -5,8 +5,7 @@
 #include <stdio.h>
 #include <sys/time.h>
 #include <cstring>
-
-
+#include <cxxabi.h>
```

Comment:
This would likely break the Windows build, along with other compilers that may not support the GCC ABI.  You should protect this include and use of the GCC ABI with the appropriate macro.  Best practice would be to wrap `abi::__cxa_demangle` in a function that returns the original string if the GCC ABI is not available.  Here is an example:

https://github.com/trilinos/Trilinos/blob/master/packages/teuchos/core/src/Teuchos_TypeNameTraits.cpp

Reply:
The example doesn't show the right macro, but that shouldn't be too hard to figure out :-) .

Reply:
Indeed it would be easy to change, but since the library is only built through a simple Makefile (no CMake or configuration AFAIK), it is not clear to me how to make it in a clean way.
Should I just specify an optional macro in the Makefile and add a Readme file for this particular timer?

Reply:
Yeah just add a define in the Makefile. Make it do demangle by default. I assume the demangle function returns the original string if it couldn't do it right? Because often it can't demangle for example because of using lambdas where the names are not standardized.